resolwe_bio/processes/genome_browser/UCSC_track_hub.yml

```diff
@@ -0,0 +1,75 @@
+# ==============
+# UCSC track hub
+# ==============
+---
+
+- slug: genomebrowser-macs14
+  name: UCSC track hub
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: "resolwebio/chipseq:1.1.0"
+    resources:
+      network: true
+  data_name: "UCSC track hub ({{ name }})"
+  version: 0.0.1
+  type: data:genomebrowser:all:bigwig:macs14
+  category: genomebrowser
+  persistence: CACHED
+  description: >
+    Create UCSC track hub.
+  input:
+    - name: macs
+      label: MACS1.4
+      type: list:data:chipseq:macs14
+    - name: name
+      label: Track hub name
+      type: basic:string
+    - name: email
+      label: Email address
+      type: basic:string
+      required: false
+  output:
+    - name: url_link
+      label: UCSC track hub link
+      type: basic:string
```

Comment:
We also use `basic:url:view` and `basic:url:download` which might be more appropriate here.